docs/deployment_guide.md

```diff
@@ -0,0 +1,1242 @@
+# **Confidential Computing Deployment Guide**
+
+## Documentation History
+DU-11462-001
+
+| Version | Date | Authors | Description of Change |
+| ------- | ---- | ------- | --------------------- |
+| 1.0 | 7/25/2023 | Rob Nertney | Initial Version for Early Access |
+
+## Using This Guide
+
+This guide is the most distilled set of instructions required to configure a system for Confidential Computing (CC) with the NVIDIA®Hopper™ H100 GPU. Explanations as to the value of a particular step, or the details of what is going on behind the scenes are covered in several of our other collateral, such as our whitepaper, GTC talks, and YouTube videos.  
+
+
+This guide is intended to provide instructions that are targeted to various personas who want to use Hopper Confidential Compute (HCC). These personas are rough definitions of individuals who might have different responsibilities in the overall confidential system. The overall flow of using one is illustrated in Figure 1. 
+
+**Figure 1.      Overall Workflow** 
+![Figure 1.      Overall Workflow](img/Figure1.png)
+You can see that not every person involved in enabling and using CC will be required at every step. For example, a CSP might only provision a VM, and the user then takes over.
+
+**Figure 2.      Workflow Example** 
+![Figure 2.      Workflow Example](img/Figure2.png)
+In this example, the CSP does not require a policy for how often the GPU must be checked for integrity/validity, nor does it need to consider the infrastructure requirements for Confidential Containers. The user/tenant of the CSP does not need to consider the steps required to configure the GPU for confidential or non-confidential modes. Depending on who you are, and what your goals are, you might require all, or only a fraction, of the steps. 
+
+The following personas have been defined: 
+- **Hardware IT Administrator** 
+- **Host OS Administrator** 
+- **Virtual Machine Administrator** 
+- **Virtual Machine User** 
+- **Container User**
+
+You can read the entire documentation or jump directly to the section that  most accurately describes your persona use case. This guide is organized in a linear manner, so  reading all sections in order will make logical sense to a developer who considers themselves all the above personas. 
```

Comment:
There are many instances of double spaces between words. For example `that  most` and `that  enable`.

Reply:
> There are many instances of double spaces between words. For example `that most` and `that enable`.

@steven-bellock  This should be fixed now. please review and confirm